### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by sparshk**
Added members app , added it to INTSALLED_APPS and updated urls.py. Please review.

### CODE REVIEW

**Comment by vishvanath45** [comment]
Add a field `profile_pic` in model.

**Comment by vishvanath45** [comment]
Display members in template by grouping them according to `year_of_graduation`.

File: website/members/templates/members/members_list.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html' %}
+
+{% block content %}
+	<div class="member-list">
+		<h2>Members:</h2>
+		{% for member in members %}
+			<h3>Name : {{ member.name }}</h3>
+			<h3>Year : {{ member.year }}</h3>
```
Comment (vjs22334) [inline website/members/templates/members/members_list.html]: change this to year_of_graduation

**Comment by sparshk** [comment]
Changes made as suggested. Kindly review.

**Pull request merged**